frontend/src/LandingView/styles/BottomStyled.js

```diff
@@ -7,9 +8,44 @@ const BtmContentStyles = styled.div`
 	font-weight: 100;
 	margin: 0 auto;
 	margin-top: 2%;
-	width: 40%;
+	width: 100%;
 	height: 2vh;
```

Comment:
Unless you want something to scale the entirety of the width or height of your view port, you should avoid using vw and vh due to inconsistent sizing across browsers.